mysql-test/suite/component_keyring_kmip/t/upgrade.test

```diff
@@ -0,0 +1,21 @@
+--source include/big_test.inc
```

Comment:
I accidentally added these back somehow, after removing them first - these tests do not work with KMIP, because they either assume that they can take a backup by copying the keyring file, or that they can use two different keyrings. Removed.